independent-publisher-connectors/Annature/apiProperties.json

```diff
@@ -0,0 +1,83 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_client_id": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "required": "true",
+            "tabIndex": 2
+          },
+          "description": "Annature API Client Id",
+          "displayName": "Annature Api Id",
+          "tooltip": "Specifiy Annature Api Client Id"
```

Comment:
@iddycol all files should be free of grammatical errors and typos. in the tooltip you have "Specifiy...", can you please change to "Specify..."

Reply:
Issue Fixed now.